mtls/mtls.py

```diff
@@ -900,21 +953,56 @@ def add_user(self, fingerprint, is_admin=False):
         else:
             _type = "User"
         click.secho(
-            "Added {_type}: {fingerprint}".format(
-                fingerprint=fingerprint, _type=_type
-            ),
+            f"Added {_type}: {fingerprint}",
             fg="green",
         )
 
     def remove_user(self, fingerprint, is_admin=False):
         msg = "Signing Remove User Request..."
-        payload = {
-            "type": "ADMIN" if is_admin else "USER",
-            "fingerprint": fingerprint,
-            "signature": str(self.gen_sig(fingerprint.encode("UTF-8"), msg)),
-        }
-        response = self.send_request(payload, method="post")
-        response = response.json()
+        payload = {}
+        if is_admin:
+            payload["admin"] = True
+
+        if payload:
+            signature = self.gen_sig(json.dumps(payload, sort_keys=True), msg)
+        else:
+            signature = self.gen_sig("NOCONTENT", msg)
```

Comment:
Why isn't this e.g. a signature over the empty object? `{}`

Reply:
It probably could be. This is the mechanism that is set in the server. https://github.com/drGrove/mtls-server/blob/124865fadd23dca58f74064c681fd3830aac5b59/mtls_server/auth.py#L94

Reply:
mmmm. okay.
it's weird to me that a request would only *sometimes* have a json body; rather than requiring a body of the empty object.

Reply:
We probably could get rid of that entire mechanism